user_manual/android_app.rst

```diff
@@ -200,36 +204,44 @@ gives you the option to sort your files by date, or alphabetically.
 .. figure:: images/android-6.png
    :alt: Top-right menu.   
   
-Sharing files
+Sharing Files
 ^^^^^^^^^^^^^
 
-You can share with other ownCloud users and groups, and create public share
-links. 
+You can share with other ownCloud users and groups, and create public share links. 
+
+.. note:: Multiple public links per/file is only available with ownCloud X.
+
 To share a file, you first need to either:
 
 1. Long-click its name, and click the share icon at the top of the screen 
-2. Click its name and then click  the share icon at the top of the screen
+2. Click its name and then click the share icon at the top of the screen
 
-The dialog which appears shows a list of users with whom the file is already
-shared. 
+The dialog which appears shows a list of users and groups with whom the file is already
+shared, as well as a list of one or more public links. 
 
-.. figure:: images/android-12.png
-   :alt: Sharing files.
+.. figure:: images/multiple_share_link.png
+   :alt: File Shares.
 
 From here you can:
 
 - Share the file with one or more users and groups
-- Share a link to the file via a range of options
+- Share one or more links to the file via a range of options
 - Enable password protection
 - Set a share expiration date
 
-To share the file with a new user or group, click "Add User Or Group", where
-you will be able to enter their details. 
+To share the file with a new user or group, click the plus sign next to "Users and Groups", where
+you will be able to find and add them to the share list.
+After a user or group has been added, you can adjust the editing and on-sharing options available for them.
 
 .. NOTE:: 
    If your ownCloud server administrator has enabled username auto-completion,
    when you start typing user or group names they will auto-complete. 
-   
+
+To create a public link, click the plus symbol next to "Public Links". 
+This will display the options available for that link, including "Allow editing", "Password", and "Expiration".
+After the options have been suitably configured, click "Save" to create the link.
+If you do not want to create the public link, click "Cancel".
+
```

Comment:
I think that the paragraph "You can create a Federated Share Link by entering the username and remote URL" should go above public link creation, because it is related to share with a new user or group, more than sharing via link.

Reply:
Not sure if here or above, but should be cleared that multiple public share links are only available for oC server versions 10 and newer. Anyone with an older server does not have this option available.

Reply:
@davigonz I've just re-arranged that paragraph (the change will be synced shortly). @jesmrec that's covered in the note in the Sharing Files section.

Reply:
@settermjd LGTM